### PR TITLE
Reimbursement: Hide Conversation for Initial Drafts

### PR DESCRIPTION
**Pull request by sampoder**
If you've never submitted it, we won't show the conversation view.

To implement this I've added `initial_draft?` to `Reimbursement::Expense` and then conditionally rendered the partial.

### CODE REVIEW

File: app/views/reimbursement/reports/show.html.erb [hunk 1]
```diff
@@ -104,7 +104,9 @@
 
 <%= render partial: "actions", locals: { report: @report, user: @user } %>
 
-<%= render partial: "conversation" %>
+<% unless @report.initial_draft? %>
```
Comment (garyhtou) [inline app/views/reimbursement/reports/show.html.erb]: As an organizer, I may want to comment on an old, `initial_draft?` report to ask if the user is ready to submit

Comment (sampoder) [reply app/views/reimbursement/reports/show.html.erb]: How would the user see your comment?

Comment (garyhtou) [reply app/views/reimbursement/reports/show.html.erb]: oh- good point 🤡

Comment (garyhtou) [reply app/views/reimbursement/reports/show.html.erb]: nevermind then. the logic condition for that is gonna get overly compmlicated

**Review by garyhtou — APPROVED** [review]

**Pull request merged**